docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -269,15 +269,19 @@ Fields:
     non-module related build variant fields ([context](../decisions/2024-07-18_allow_module_expansions)).
 -   `tasks`: a list of tasks to run, referenced either by task name or by tags.
     Tasks listed here can also include other task-level fields, such as
-    `batchtime`, `cron`, `activate`, `depends_on`, and `run_on`. We can also
-    [define when a task will run](#limiting-when-a-task-or-variant-will-run). If there are
+    `batchtime`, `cron`, `activate`, `depends_on`, `stepback`, and `run_on`. 
+    We can also [define when a task will run](#limiting-when-a-task-or-variant-will-run). If there are
     conflicting settings definitions at different levels, the order of priority
     is defined [here](#task-fields-override-hierarchy).
 -   `activate`: by default, we'll activate if the whole version is
     being activated or if `batchtime` specifies it should be activated. If
     we instead want to activate immediately, then set activate to true.
     If this should only activate when manually scheduled or by
     stepback/dependencies, set activate to false.
+-   `stepback`: indicate if this variant should opt-in or out of stepback. 
+    (If disabled at the project-level, this value will be ignored.)
+-   `deactivate_previous`: indicate if this variant should unschedule older 
+    mainline tasks on success (if disabled at the project-level, this value will be ignored.)
```

Comment:
Can we also mention this will override the project setting if it is set to `false` on the BV while the project ref has it set to `true`?

Reply:
Trying to avoid making this doc too verbose, but added a bit